drivers/include/apds9007.h

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2018 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_apds9007 APDS9007 light sensor
+ * @ingroup     drivers_sensors
+ * @brief       Driver for the APDS9007 light sensor
+ *
+ * The connection between the MCU and the APDS9007 is based on the
+ * GPIO-interface.
+ *
+ * @{
+ *
+ * @file
+ * @brief       Driver for the APDS9007 light sensor
+ *
+ * @author      Hyung-Sin Kim <hs.kim@cs.berkeley.edu>
+ */
+
+#ifndef APDS9007_H
+#define APDS9007_H
+
+#include <stdint.h>
+#include "periph/gpio.h"
+#include "periph/adc.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Stabilization time for APDS9007 output
+ */
+#ifndef APDS9007_STABILIZATION_TIME
+#define APDS9007_STABILIZATION_TIME 20000UL
+#endif
+
+/**
+ * @brief   APDS9007 specific return values
+ */
+enum {
+    APDS9007_OK      = 0,    /**< everything went as expected */
+    APDS9007_ADCERR = -1,    /**< ADC reading error */
+};
+
+/**
+ * @brief   Parameters needed for device initialization
+ */
+typedef struct {
+    gpio_t    gpio;            /**< GPIO pin that sensor is connected to */
```

Comment:
The data sheet calls this pin 'shutdown' which also makes sense given it's function.